.github/workflows/clang-format-check.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 concurrency:
   # If workflow for PR or push is already running, stop it and start a new one.
-  group: build-${{ github.ref }}
+  group: clang-format-${{ github.ref }}
```

Comment:
Makes the clang-format-check workflow identifier different than the pipeline workflow identifier, so one flow doesn't cancel the other.